dtcli/scripts/dt.py

```diff
@@ -170,5 +170,32 @@ def build(**kwargs):
     )
 
 
+@extension_dev.command(
+    help="comand packs python package as a datasource. It uses pip to download all dependencies and create whl files"

```

Comment:
I'd describe that simply with "builds Python extension".

Reply:
I was thinking about it, but we have "dt ext build" command, which creates whole extension pack, and I decided to not use this name